src/rules/property-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,96 @@
+import { testRule } from "../../../testUtils"
+import rules from "../../../rules"
+import { ruleName, messages } from ".."
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [true],
+
+  accept: [ {
+    code: ".foo { color: green; }",
+  }, {
+    code: ".foo { COLoR: green; }",
+  }, {
+    code: ".foo { fill: black; }",
+  }, {
+    code: ".foo { -webkit-align-self: center; }",
+  }, {
+    code: ".foo { align-self: center; }",
+  }, {
+    code: ".foo { --bg-color: white; }",
+    description: "ignore standard CSS variables",
+  }, {
+    code: ".foo { *width: 100px; }",
```

Comment:
Can we get a companion reject test for this that shows that only the hack part (e.g. the `*`) is ignored, but the property is still tested e.g.:

```js
{
  code: ".foo { *widh: 100px; }",
  message: messages.rejected("widh"),
  line: 1,
  column: 9,
}
```